test/e2e/tests/e2e_test.go

```diff
@@ -197,6 +197,11 @@ func fetchBlockChain(t *testing.T) []*types.Block {
 	return blocks
 }
 
+type ABCIExecution struct {
+	Type string
+	Reqs []*abci.Request
+}
```

Comment:
Where is this struct used?

Reply:
It is not used. It was used in one of the previous versions. I will remove it. Thanks!